### PR TITLE
Add DISABLE_SPRING=1 to README

### LINKED ISSUE #431: uninitialized constant Administrate::Generators::InstallGenerator::DashboardManifest
Hi :smiley: 

When I run the installer for Administrate, I get the following error:

```
rails generate administrate:install
Running via Spring preloader in process 33128
      create  app/controllers/admin/application_controller.rb
       route  namespace :admin do
    DashboardManifest::DASHBOARDS.each do |dashboard_resource|
      resources dashboard_resource
    end

    root controller: DashboardManifest::ROOT_DASHBOARD, action: :index
  end
      create  app/dashboards/dashboard_manifest.rb
/usr/local/lib/ruby/gems/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:44:in `manifest': uninitialized constant Administrate::Generators::InstallGenerator::DashboardManifest (NameError)
	from /usr/local/lib/ruby/gems/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:36:in `dashboard_resources'
	from /usr/local/lib/ruby/gems/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:32:in `singular_dashboard_resources'
	from /usr/local/lib/ruby/gems/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:24:in `run_dashboard_generators'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /usr/local/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/generators.rb:157:in `invoke'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /usr/local/lib/ruby/gems/2.3.0/gems/railties-4.2.5/lib/rails/commands.rb:17:in `<top (required)>'
	from /Users/frabrunelle/administrate/bin/rails:9:in `<top (required)>'
	from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

Is anyone else getting this error? I get this error even on a totally new Rails application, so it should be pretty easy to reproduce.

### PR DESCRIPTION
**Pull request by buren**
Closes #431

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ gem "administrate", "~> 0.1.3"
 Re-bundle, then run the installer:
 
 ```bash
-$ rails generate administrate:install
+$ DISABLE_SPRING=1 rails generate administrate:install
```
Comment (s2t2) [inline README.md]: :clap:

**Comment by johnpaulashenfelter** [comment]
:+1: This *really* needs to get int the README so the first experience with this is a bit smoother than looking through the issues.

**Comment by coreyward** [comment]
This doesn't work for me. I still get the following error:

```
ruby-2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:44:in `manifest': uninitialized constant Administrate::Generators::InstallGenerator::DashboardManifest (NameError)
```

This is after running `spring stop`, exiting Guard, restarting Pow, and killing any remaining Ruby processes in Activity Monitor.

**Comment by buren** [comment]
@coreyward I had the same problem trying to install administrate on another project where neither `DISABLE_SPRING=1` or `spring stop` worked. This PR seems however to fix that issue: https://github.com/thoughtbot/administrate/pull/450. I added the code manually to the gem temporarily and got administrate to install. (The [tests for that PR doesn't pass](https://circleci.com/gh/thoughtbot/administrate/1156), but that seems related to the recent Rails security advisories..)

**Comment by graysonwright** [comment]
This should be fixed by #464. Thanks!

**Pull request closed without merging**